### PR TITLE
LPS-131619 SF, rename

### PR DESCRIPTION
**Pull request by ling-alan-huang**

### CODE REVIEW

**Comment by ling-alan-huang** [comment]
Hey @joseabelenda
I did SF changes in order to pass `ci:test:sf`, can you help to check if thoes break anything?

**Comment by ling-alan-huang** [comment]
Thanks.

**Comment by ling-alan-huang** [comment]
ci:test:sf

**Comment by ling-alan-huang** [comment]
ci:test:relevant

File: modules/apps/digital-signature/digital-signature-impl/src/main/java/com/liferay/digital/signature/internal/configuration/DigitalSignatureConfiguration.java [hunk 1]
```diff
@@ -27,13 +27,14 @@
...
-	localization = "content/Language", name = "digital-signature-configuration"
+	localization = "content/Language",
+	name = "digital-signature-configuration-name"
 )
 public interface DigitalSignatureConfiguration {
 
 	public boolean enable();
 
-	public String apiUsername();
+	public String apiUserName();
```
Comment (joseabelenda) [inline modules/apps/digital-signature/digital-signature-impl/src/main/java/com/liferay/digital/signature/internal/configuration/DigitalSignatureConfiguration.java:37]: Hi @ling-alan-huang.
Isn't necessary.
Was added suppress CamelCaseNameCheck in modules/apps/digital-signature/source-formatter-suppressions.xml and merged today.
We need to use apiUserName instead apiUsername.

File: modules/apps/digital-signature/digital-signature-impl/src/main/java/com/liferay/digital/signature/internal/configuration/DigitalSignatureConfigurationForm.java [hunk 2]
```diff
@@ -63,7 +63,7 @@
 	public String apiAccountId();
 
 	@DDMFormField(label = "%api-username")
-	public String apiUsername();
+	public String apiUserName();
```
Comment (joseabelenda) [inline modules/apps/digital-signature/digital-signature-impl/src/main/java/com/liferay/digital/signature/internal/configuration/DigitalSignatureConfigurationForm.java:66]: Same from above.

File: modules/apps/digital-signature/digital-signature-impl/src/main/java/com/liferay/digital/signature/internal/http/DSHttp.java [hunk 3]
```diff
@@ -60,7 +60,7 @@ private String _getDocuSignAccessToken(
 		throws Exception {
 
 		JSONObject jsonObject = DSAccessTokenWebCacheItem.get(
-			digitalSignatureConfiguration.apiUsername(),
+			digitalSignatureConfiguration.apiUserName(),
```
Comment (joseabelenda) [inline modules/apps/digital-signature/digital-signature-impl/src/main/java/com/liferay/digital/signature/internal/http/DSHttp.java:63]: Same from above.

File: modules/apps/digital-signature/digital-signature-impl/src/main/resources/content/Language.properties [hunk 4]
```diff
@@ -3,6 +3,6 @@ api-account-id=API Account ID
 api-username=API Username
 category.digital-signature=Digital Signature
 digital-signature=Digital Signature
-digital-signature-configuration=Digital Signature Configuration
+digital-signature-configuration-name=Digital Signature Configuration
```
Comment (joseabelenda) [inline modules/apps/digital-signature/digital-signature-impl/src/main/resources/content/Language.properties:6]: I'll change this in next PR.
We need to change in DigitalSignatureConfiguration.java too.

**Review by joseabelenda — COMMENTED** [review]
Hi @ling-alan-huang .
Thanks for PR.
See my comments, please.

**Comment by ling-alan-huang** [comment]
Thanks @joseabelenda

**Pull request closed without merging**